tests/e2e/ColorPicker.spec.ts

```diff
@@ -0,0 +1,142 @@
+import modePage from "./pages/modePage";
+import colorFormatPage from "./pages/colorFormatPage";
+
+fdescribe("Color Picker", () => {
+
+    beforeAll(() => {
+        modePage.open();
+    });
+
+    it("should render a button with a background: rgb(76, 175, 80)", () => {
+        modePage.button.waitForVisible();
+        modePage.buttonColor.waitForVisible();
+
+        const buttonBackgroundColor = browser.getCssProperty(".mx-name-colorPicker3 .widget-color-picker-inner", "background").value;
+        expect(buttonBackgroundColor).toContain("rgb(76,175,80)");
+    });
+
+    it("should render an input box with a color value: #4caf50", () => {
+        modePage.inPutBoxTab.click();
+        modePage.inPutBox.waitForVisible();
+
+        const inPutBoxValue = browser.getValue(".mx-name-colorPicker17 input");
+        expect(inPutBoxValue).toContain("#4caf50");
+    });
+
+    it("should be rendered inline", () => {
+        modePage.inlineTab.click();
+        modePage.inLineColorPicker.waitForVisible();
+
+        const inPutBoxValue: boolean = browser.isExisting(".mx-name-colorPicker27 .sketch-picker ");
+        expect(inPutBoxValue).toBe(true);
+    });
+
+    it("should be render type: Sketch", () => {
+
```

Comment:
remove empty line , for the next 12 tests

Reply:
I think its only for this one... the setup.. execution and expects are three steps that's what i remember